pages/common/chronic.md

```diff
@@ -0,0 +1,16 @@
+# chronic
+
+> Display `stdout` and `stderr` of a command if and only if it fails.
```

Comment:
Sorry for the ignorance, but why "If and only if"? Afaik it is a logical-mathematical statement that means if A then B, if B then A.

Reply:
Yes, it prints the output if the command fails and does not print the output if the command doesn't fail.